src/main/java/com/artipie/VertxMain.java

```diff
@@ -194,9 +195,10 @@ private void startRepos(final Settings settings, final Metrics metrics) {
         final String yaml = ".yaml";
         final Collection<RepoConfig> configs = storage.list(Key.ROOT).thenApply(
             keys -> keys.stream().map(Key::string)
-                .filter(name -> name.endsWith(yaml) && name.charAt(0) != '_')
-                .map(name -> name.substring(0, name.length() - yaml.length()))
-                .map(name -> new RepositoriesFromStorage(storage).config(name))
+                .filter(name -> name.charAt(0) != '_')
+                .map(name -> new ConfigFilesExtensions.Trim(name).value())
```

Comment:
@genryxy `ConfigFilesExtensions.Trim` this class name does not look like name of an object, but looks more like a verb. I'd suggest to try come up with some better design